### PR TITLE
Add shuffler.fm to list of supported players

### PR DESCRIPTION
**Pull request by imjennyli**
Resolves Issue #57

### CODE REVIEW

File: json/shuffler_fm.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+    "artUrl" : "",
+    "artist": "$('.artist-name').text()",
+    "track": "$('.track-title').text()",
+    "currentTime": "$('.elapsed').text()",
+    "remainingTime": "$('.remaining').text()",
+
+    "COMMENT_4": "Finally, this block contains info on how to perform actions",
+
+    "click_play": "$('[id=play-pause]').click()",
```
Comment (kkamperschroer) [inline json/shuffler_fm.json]: This is nit-picky of me, but JQuery has a `$('#play-pause')` where `#` means id. Could you use that syntax instead to be consistent with the other players where an `id` selector is available?

Comment (imjennyli) [inline json/shuffler_fm.json]: Sure! I'm new to jQuery so I wasn't super sure what the difference was with the other. I'll update tonight.

**Comment by imjennyli** [comment]
OH! Whoops. The comment spanned the whole block, and my eyes just caught the first part. That totally makes more sense. Thanks @NasaGeek

**Comment by imjennyli** [comment]
Closing this, I found an issue with the button clicks in the pop-up. D'oh!

**Comment by kkamperschroer** [comment]
If it's double clicking, it's probably not a real issue. Sometimes the
content script gets doubly injected when reloading extension updates and
things can get weird. Try closing and reopening the tab with the player and
see if the issue disappears.

**Comment by imjennyli** [comment]
I'm in a strange state where sometimes the popup evaluates the play/pause state of the player, but the popup doesn't change at all to indicate if it is playing, or what's playing. I'll investigate sometime next week.

**Pull request closed without merging**